benchmarks/WolfSheep/wolf_sheep.py

```diff
@@ -94,26 +93,42 @@ class GrassPatch(CellAgent):
     A patch of grass that grows at a fixed rate and it is eaten by sheep
     """
 
-    def __init__(self, unique_id, model, fully_grown, countdown):
+    @property
+    def fully_grown(self):
+        return self._fully_grown
+
+    @fully_grown.setter
+    def fully_grown(self, value: bool) -> None:
```

Comment:
This feels duplicate with the part below somehow. Why do we need these properties and setters?

Reply:
it is not a nice solution I agree. The issue is that events take a callable, so I used the set_fully_grown function for this. It can be done cleaner by using `setattr` and `function_args`.